README.md

```diff
@@ -8,7 +9,7 @@ layers of the Agoric platform, with
 providing the lower layers.
 If you want to build on top of this platform, you don't need these
 repositories: instead you should
-[follow our instructions for getting started](https://agoric.com/documentation/getting-started/)
+[follow our instructions for getting started](https://docs.agoric.com/guides/getting-started/)
```

Comment:
@Tyrosine22 let's please fix these links on the other end too with a redirect. Who knows how many other links there are to the old URL. (IOU an issue in the docs repo)